src/components/v2/V2Create/tabs/OthersTabContent.tsx

```diff
@@ -0,0 +1,130 @@
+    dispatch(editingV2ProjectActions.setBallot(fields.ballot))
+  }, [dispatch, othersForm])
+
+  const resetOthersForm = useCallback(() => {
+    othersForm.setFieldsValue({
+      pausePay: fundingCycleMetadata?.pausePay ?? false,
+      pauseMint: fundingCycleMetadata?.pauseMint ?? false,
+      ballot: fundingCycleData?.ballot ?? ballotStrategies()[2].address, // 3-day delay default
+    })
+  }, [
+    fundingCycleMetadata?.pausePay,
+    fundingCycleMetadata?.pauseMint,
+    fundingCycleData?.ballot,
+    othersForm,
+  ])
+
+  // initially fill form with any existing redux state
+  useEffect(() => {
+    resetOthersForm()
+  }, [resetOthersForm])
+
+  return (
+    <div>
+      <Space direction="vertical" size="large">
+        <p>
+          <Trans>
+            <strong>Note: </strong>Updates you make to attributes in this
+            section will <i>not be applied immediately</i> and only take effect
+            in <i>upcoming funding cycles.</i>
+          </Trans>
+        </p>
+        <Form form={othersForm} layout="vertical">
+          <Form.Item
+            name="pausePay"
+            label={t`Pause payments`}
+            extra={t`Your project cannot receive direct payments while paused.`}
+            valuePropName={'checked'}
+            style={{ ...shadowCard(theme), padding: '2rem' }}
+          >
+            <Switch />
+          </Form.Item>
+          <Form.Item
+            name="pauseMint"
+            label={t`Allow minting tokens`}
+            extra={t`Enabling this allows the project owner to manually mint any amount of tokens to any address.`}
+            valuePropName="checked"
+            style={{ ...shadowCard(theme), padding: '2rem' }}
+          >
+            <React.Fragment>
+              <Switch
+                onChange={val => {
+                  setShowMintingWarning(val)
+                }}
+              />
+              {showMintingWarning && (
+                <Form.Item>
+                  <div style={{ color: theme.colors.text.warn, marginTop: 10 }}>
+                    <Trans>
+                      <strong>Note: </strong>Enabling tokens to be minted will
+                      appear risky to contributors, and should only be used when
+                      necessary.
+                    </Trans>
+                  </div>
+                </Form.Item>
+              )}
+            </React.Fragment>
+          </Form.Item>
+          <FormItems.ProjectReconfiguration
+            value={
+              othersForm.getFieldValue('ballot') ?? fundingCycleData?.ballot
+            }
+            onChange={(address: string) =>
+              othersForm.setFieldsValue({ ballot: address })
+            }
+            style={{ ...shadowCard(theme), padding: '2rem' }}
+          />
+          <Form.Item>
+            <Button
+              htmlType="submit"
+              type="primary"
+              onClick={async () => {
+                await othersForm.validateFields()
```

Comment:
**suggestion** we can use the `onFinish` handler of the `Form` element, instead of the button click, and `validateFields` is called for us automatically :). Check the other forms for an example.

Reply:
Sweet, nice! Thanks